rero_ils/modules/operation_logs/permissions.py

```diff
@@ -32,11 +32,11 @@ def list(cls, user, record=None):
         :param record: Record to check.
         :return: "True" if action can be done.
         """
-        # all users may list operation_log records.
-        # TODO: check with group PO here.
+        # all users (lib, sys_lib) can read operation_log records.
         if not current_patron:
             return False
-        return True
+        if current_patron.is_librarian:
+            return True
```

Comment:
`return current_patron.is_librarian`.
With this implementation, a user with only "patron" role, this function return `None` instead False